src/Init/System/IO.lean

```diff
@@ -360,6 +361,9 @@ Will succeed even if no lock has been acquired.
 -/
 @[extern "lean_io_prim_handle_unlock"] opaque unlock (h : @& Handle) : IO Unit
 
+/-- Returns true if a handle refers to a Windows console or Unix terminal. -/
+@[extern "lean_io_prim_handle_isatty"] opaque isatty (h : @& Handle) : BaseIO Bool
```

Comment:
Since the underlying function is platform-dependent anyway, why not a more standard name like `isTty` or `isTerminal`? https://docs.rs/termion/latest/termion/fn.is_tty.html